packages/strapi-plugin-users-permissions/services/User.js

```diff
@@ -63,7 +63,11 @@ module.exports = {
    * @return {Promise}
    */
   fetch(params, populate) {
-    return strapi.query('user', 'users-permissions').findOne(params, populate);
+    let defaultPopulate = ['role'];
```

Comment:
You can simplify this function with

fetch(params, populate = ['role']) {
  return strapi.query('user', 'users-permissions').findOne(params, populate);
}

Reply:
Yup, I mentioned in my previous comment hehe